### PR TITLE
Updating SQL Server on Linux Management - named pipes

### PR DESCRIPTION
**Pull request by clayton006**
Adding information to document to clarify that named pipes connectivity is not possible for SQL on Linux.

### CODE REVIEW

**Comment by PRMerger13** [comment]
@clayton006 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/linux/sql-server-linux-management-overview.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ Almost everything you can do in a client tool can also be accomplished with Tran
 
 The new Azure Data Studio is a cross-platform tool for managing SQL Server. For more information, see [Azure Data Studio](../azure-data-studio/what-is-azure-data-studio.md).
 
+## Named Pipes
```
Comment (amvin87) [inline docs/linux/sql-server-linux-management-overview.md:31]: Can we phrase this as " Named pipes protocol is not supported for SQL Server on Linux."

**Comment by PRMerger7** [comment]
@clayton006 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by VanMSFT — APPROVED** [review]

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**